eclass/ecm.eclass

```diff
@@ -674,7 +707,7 @@ ecm_src_install() {
 		fi
 	}
 
-	if [[ -n ${_KDE_ORG_ECLASS} && -d "${ED}"/usr/share/metainfo/ ]]; then
+	if [[ -n ${_KDE_ORG_ECLASS} ]] && [[ -d "${ED}"/usr/share/metainfo/ ]]; then
```

Comment:
The old one should be a tiny bit faster and is more idiomatic bash (but combining like that isn't valid for POSIX `[`).

Reply:
so ... I'll leave it as it is right now?

Reply:
I'd prefer it merged together but it's not worth you rebasing.